app/helpers/application_helper/listnav.rb

```diff
@@ -216,20 +216,32 @@ def valid_html_id(id)
       id
     end
 
+    NO_ACCORDION = %w[chargeback_assignment
+                       chargeback_rate
+                       chargeback_report].to_set.freeze
+
     # Create a collapsed panel based on a condition
     def miq_accordion_panel(title, condition, id, &block)
       id = valid_html_id(id)
-      content_tag(:div, :class => "panel panel-default") do
-        out = content_tag(:div, :class => "panel-heading") do
-          content_tag(:h4, :class => "panel-title") do
-            link_to(title, "##{id}",
-                    'data-parent' => '#accordion',
-                    'data-toggle' => 'collapse',
-                    :class        => condition ? '' : 'collapsed')
+      if NO_ACCORDION.include?(@layout)
```

Comment:
Its semantically not correct to have a `NO_ACCORDION` inside a `miq_accordion_panel`. Why not just not call the function if there are no accordions?